js/content/common.js

```diff
@@ -212,6 +212,18 @@ let ExtensionLayer = (function() {
         script.parentNode.removeChild(script);
     };
 
+    // NOTE: use cautiously!
+    // Load remote script in the context of the current tab
+    self.loadInPageContext = function(fun) {
+        return new Promise(function(res) {
+            let script  = document.createElement("script");
```

Comment:
I would be happier if `res` was named `resolve` like every other `new Promise()` in the extension. (And it would be nice to include `reject` even if you don't call it as a readability measure.